### DISCUSSION TITLE
Upgrade cloud-sdk docker image tag set in a file

### DISCUSSION THREAD

**Discussion by chamoulox** [renovatebot/renovate#21219] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

Bitbucket  server v8.4.1

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello, 

I'm using [cloud-sdk ](https://hub.docker.com/r/google/cloud-sdk/) docker image and I want to keep it up to date with renovate.

As I'm using it in several files, I have another file (let's say __constants.groovy__) with the tag

`CLOUD_SDK_VERSION= '420.0.0'`

In my _renovate.json_ file, I'm trying to use regexManagers to catch and update the tag.

As output, I would like a PR that changes CLOUD_SDK_VERSION to 424.0.0 (the latest one)

### What you tried so far.

So far, I tried different regex:

```
 "regexManagers": [
    {
      "description": "Update CLOUD_SDK_VERSION in Constants.groovy",
      "fileMatch": [
        "^Constants\\.groovy$"
      ],
      "matchStrings": [
        "CLOUD_SDK_VERSION = '(?<currentValue>.*?)'"
      ],
      "datasourceTemplate": "docker",
      "depNameTemplate": "google/cloud-sdk",
      "extractVersionTemplate": "^google\/cloud-sdk:(?<version>.*)$"
    }
  ]
``` 
This regex matches the basic pattern of docker tags. But it didn't work.

With DEBUG mode, I found 
```
DEBUG: getLabels(https://index.docker.io, google/cloud-sdk, latest) (repository=rd/pipeline)
DEBUG: getManifestResponse(https://index.docker.io, google/cloud-sdk, latest, get) (repository=rd/pipeline)
DEBUG: Found no results from datasource that look like a version (repository=rd/pipeline, dependency=google/cloud-sdk)
       "result": {
         "registryUrl": "https://index.docker.io",
         "releases": [],
         "changelogUrl": "https://cloud.google.com/sdk/docs/release-notes"
       }
```
So I supposed it used cloud.google.com page to fetch tags ? I then updated my regex to 
`"extractVersionTemplate": "^(?<version>.*) \\(\\d{4}-\\d{2}-\\d{2}\\)$"`

I'm a bit lost and I don't know where to look for and/or what to put to make it work. Maybe I didn't choose the easiest/recommanded way.

Thanks for the help !

**Comment by viceice** [comment, marked as answer]
remove `extractVersionTemplate`

**Comment by chamoulox** [comment]
Thank you !
